src/lazy/components/client.tsx

```diff
@@ -35,19 +35,44 @@ export function createComponentClient<C extends ComponentType<any>>({
   return (props: ComponentProps<C>) => {
     const { setFallback } = useContext(LazySuspenseContext);
     const [, setState] = useState();
-    const isOwnPhase = usePhaseSubscription(defer);
 
-    useMemo(() => {
-      if (isOwnPhase) {
+    if (defer !== PHASE.LAZY) {
+      const isOwnPhase = usePhaseSubscription(defer);
+
+      useMemo(() => {
+        if (isOwnPhase) {
+          deferred.start().catch((err: Error) => {
+            // Throw the error within the component lifecycle
+            // refer to https://github.com/facebook/react/issues/11409
+            setState(() => {
+              throw new LoaderError(moduleId, err);
```

Comment:
Now that I saw this, I've created https://github.com/atlassian-labs/react-loosely-lazy/issues/66